packages/suite/src/components/suite/modals/ReduxModal/DeviceContextModal/PassphraseWalletConfirmationStep2.tsx

```diff
@@ -0,0 +1,55 @@
+import { Button, Text, Icon, Warning } from '@trezor/components';
+import { Translation } from 'src/components/suite/Translation';
+import { PassphraseHeading } from './PassphraseHeading';
+import { PassphraseList, PassphraseItem } from './PassphraseList';
+import { ContentType } from './types';
+
+type PassphraseWalletConfirmationStep2Props = {
+    setContentType: React.Dispatch<React.SetStateAction<ContentType>>;
+};
+
+export const PassphraseWalletConfirmationStep2 = ({
+    setContentType,
+}: PassphraseWalletConfirmationStep2Props) => {
+    return (
```

Comment:
nit: inline return `=> (<> ....)` with no `{` and no `return`

I thought we have a lint for this. :thinking: